topostats/grains.py

```diff
@@ -277,12 +279,16 @@ def find_grains(self):
                 self.directions[direction]["labelled_regions_02"] = self.label_regions(
                     self.directions[direction]["removed_small_objects"]
                 )
-                self.get_region_properties(self.directions[direction]["labelled_regions_02"])
-
+                self.region_properties[direction] = self.get_region_properties(
+                    self.directions[direction]["labelled_regions_02"]
+                )
+                LOGGER.info(f"[{self.filename}] : Region properties calculated ({direction})")
+                print(f"#### {direction} region_properties : {self.region_properties[direction]}")
```

Comment:
Line 286 in `grains.py` still has the print statement meaning the following still appears:
```
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : Labelling Regions
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : Region properties calculated (upper)
#### upper region_properties : [<skimage.measure._regionprops.RegionProperties object at 0x156891f10>, <skimage.measure._regionprops.RegionProperties object at 0x156891b20>, <skimage.measure._regionprops.RegionProperties object at 0x156891ee0>, <skimage.measure._regionprops.RegionProperties object at 0x156891c40>, <skimage.measure._regionprops.RegionProperties object at 0x156891d60>, <skimage.measure._regionprops.RegionProperties object at 0x156891d30>, <skimage.measure._regionprops.RegionProperties object at 0x156891cd0>, <skimage.measure._regionprops.RegionProperties object at 0x156891ca0>, <skimage.measure._regionprops.RegionProperties object at 0x156891c70>, <skimage.measure._regionprops.RegionProperties object at 0x156891be0>, <skimage.measure._regionprops.RegionProperties object at 0x156891bb0>, <skimage.measure._regionprops.RegionProperties object at 0x156891f40>, <skimage.measure._regionprops.RegionProperties object at 0x156891fa0>, <skimage.measure._regionprops.RegionProperties object at 0x156891b80>, <skimage.measure._regionprops.RegionProperties object at 0x156891b50>, <skimage.measure._regionprops.RegionProperties object at 0x156891af0>, <skimage.measure._regionprops.RegionProperties object at 0x156891ac0>, <skimage.measure._regionprops.RegionProperties object at 0x156891a90>, <skimage.measure._regionprops.RegionProperties object at 0x156891a30>, <skimage.measure._regionprops.RegionProperties object at 0x156891a00>, <skimage.measure._regionprops.RegionProperties object at 0x1568919d0>]
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : Coloured regions
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : Extracted bounding boxes (upper)
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : *** Grain Statistics ***
[Wed, 13 Jul 2022 11:39:17] [INFO    ] [topostats] [minicircle] : Processing grain: 0
```
Other than that it seems good to go!